### DISCUSSION TITLE
Integrations run tasks in debug mode

### DISCUSSION THREAD

**Discussion by vdudejon** [semaphoreui/semaphore#2013] [Q&A] [answered]
I am starting to work with Integrations, and it looks like the tasks always run in debug mode.  Is there a way to run the task without it?

**Comment by fiftin** [comment, marked as answer]
Hi @vdudejon it is a bug. Will be fixed in next beta.

**Reply by vdudejon** [reply]
Thanks!

**Reply by fiftin** [reply]
Fixed in beta: https://github.com/semaphoreui/semaphore/releases/tag/v2.9.92-beta
